plugins/woocommerce-admin/client/marketplace/utils/functions.tsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * Internal dependencies
+ */
+import { Product } from '../components/product-list-content/product-list-content';
+
+interface ProductCategory {
+	id: number;
+	title: string;
+	items: Product[];
+}
+
+// Fecth data for the discover page from the WooCommerce.com API
+const fetchDiscoverPageData = async (): Promise< ProductCategory > => {
+	const fetchUrl =
+		'https://woocommerce.com/wp-json/wccom-extensions/2.0/featured';
+
+	return fetch( fetchUrl )
```

Comment:
WordPress offers `apiFetch`, which is a wrapper around `fetch`. I don't know which one we should use here, considering this endpoint is not authenticated. However, `apiFetch` is used all around WooCommerce Admin.